packages/boba/contracts/contracts/LP/L2LiquidityPool.sol

```diff
@@ -636,6 +642,53 @@ contract L2LiquidityPool is CrossDomainEnabled, ReentrancyGuardUpgradeable, Paus
         );
     }
 
+    /*
+     * Rebalance LPs
+     * @param _amount token amount that we want to move from L2 to L1
+     * @param _tokenAddress L2 token address
+     */
+    function rebalanceLP(
+        uint256 _amount,
+        address _tokenAddress
+    )
+        external
+        payable
+        onlyOwner()
+        whenNotPaused()
+    {
+        require(_amount != 0, "Amount Incorrect");
+
+        PoolInfo storage pool = poolInfo[_tokenAddress];
+
+        require(L1LiquidityPoolAddress != address(0), "L1 Liquidity Pool Not Register");
+        require(pool.l2TokenAddress != address(0), "Token Address Not Register");
+
+        if (_tokenAddress == Lib_PredeployAddresses.OVM_ETH) {
+            require(_amount <= address(this).balance, "Failed to Rebalance LP");
+            L2StandardBridge(Lib_PredeployAddresses.L2_STANDARD_BRIDGE).withdrawTo(
+                _tokenAddress,
+                L1LiquidityPoolAddress,
+                _amount,
+                DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS,
```

Comment:
the `DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS` generally is an approximation of the clientL1Pay() logic, so we should be careful if we ever want to configureGas() to update this value
I was thinking using a number passed through the params, but then that could be prone to error, so this should be good? maybe with a caution?

Reply:
It's probably ok.

Reply:
as long as `DEFAULT_FINALIZE_WITHDRAWAL_L1_GAS` is enough gas